cypress/integration/teacher-training-adviser.spec.js

```diff
@@ -2417,8 +2417,8 @@ describe('Feature - Get an adviser : Tests execution date and time : ' + new Dat
 		cy.selectWhichClassIsYourDegree('First class');
 		cy.selectStage('Secondary');
 
-		cy.doYouHaveGrade4CorAboveInEnglishAndMathsGCSEsorEuivalent('No', 'Secondary');
-		cy.areYouPlanningToRetakeEitherEnglishorMathsorBothGCSEsorEquivalent('Yes', 'Secondary');
+		cy.doYouHaveGrade4CorAboveInEnglishAndMathsGCSEsorEuivalent('No');
```

Comment:
Typo in this method name. I'd be inclined to simplify it a bit to something like:

```
cy.gcseMathsAndEnglish(false);
```

Reply:
Resolved